packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.tsx

```diff
@@ -171,6 +178,35 @@ export class GeovEntityPropsByPredicate {
     }
   }
 
+  componentDidLoad() {
+    const shadowRoot = this.hostElement.shadowRoot;
+    this.resizePage(shadowRoot);
+    window.addEventListener('resize', this.handleResize);
+  }
+
+  resizePage(shadowRoot: ShadowRoot) {
+    // Display Expand button if need. The page must be loaded in order to have the measurements
+    if (shadowRoot) {
+      const literalContainer = shadowRoot.querySelector('.literal-container');
```

Comment:
Stencil has a method to get elements more safely: https://stenciljs.com/docs/templating-jsx#getting-a-reference-to-a-dom-element